### ISSUE TITLE
Feature Request: multiple chromosomes

### ISSUE THREAD

**Issue by jpuritz** [JimWhiting91/genotype_plot#6]
It would be great to be able to look at data from multiple chromosome simultaneously.  This would be especially great for RRL applications where *de novo* assemblies are not very contiguous.

**Comment by JimWhiting91** [comment]
I agree I think that would be useful. What's the thinking in terms of how it might be presented? Would what you have in mind look something like the single outputs we have but organised into a grid format with some common axes and labelling, I guess a bit like a facet_wrap layout? Or is there something else you have in mind? My immediate thought on this is to allow the chr, start, and end flags to accept vectors or multiple regions, and in these cases assemble the outputs into a grid within the function. Does that sound like the sort of thing you have in mind?
